errata_tool/connector.py

```diff
@@ -6,18 +6,38 @@
 import re
 import time
 import six
+import krbV
```

Comment:
Does this work fine in a virtualenv?

Reply:
setup.py and errata-tool.spec will need to know about this

Reply:
setup.py has it - errata-tool.spec will need to be updated.

Reply:
While python-krbV seems to work via pip-install for 2.7, it fails spectacularly for Python 3.5/3.6. :/ I'll look again at different options, in case this isn't going to be fixed.